torch/testing/_internal/distributed/rpc/jit/rpc_test.py

```diff
@@ -249,3 +249,14 @@ def rref_script_annotation(rref_var):
 
         res = rref_script_annotation(rref_var)
         self.assertEqual(res, torch.ones(2, 2) + 1)
+
+    @dist_init
+    def test_local_rref_creation_with_ivalue(self):
+
+        # create a local RRef that holds a IValue
+        rref_local_script_class = rpc.RRef(MyScriptClass())
+        self.assertEqual(rref_local_script_class.to_here().a, 10)
+
+        # create a local RRef that holds a ScriptModule
+        rref_local_script_mod = rpc.RRef(MyScriptModule(3)._c)
```

Comment:
So users have to write "._c" to construct script module, right?

Reply:
yeah, this will give the jit_type of the script module locally. We can infer the scriptModule `py::object` directly to get its type, but I think that overkills the purpose of this PR. I will leave that with separate PR that enabling complete ScriptModule and ModuleInterface rref creation across workers. cc @xush6528